msetnxex.lua

```diff
@@ -0,0 +1,11 @@
+local setExCount = 0
+if redis.call("msetnx", unpack(KEYS)) == 1 then
+    for i, key in ipairs(KEYS) do
+        if i % 2 == 1 then
+            if redis.call("pexpire", key, ARGV[1]) == 1 then
+                setExCount = setExCount + 1               
```

Comment:
UPD: remembered the case when key could hyphotetically be gone - if it would expire while this loop is running.

But found this: https://stackoverflow.com/questions/49158805/is-this-redis-lua-script-that-deals-with-key-expire-race-conditions-a-pure-funct

Basically, ppl say that key can expire, but won't be removed while script runs.

So my initial `error(...)` instead of softer expire-success counting is still valid.

Reply:
True, its more straight forward to return true/false or success/error statement instead of a count.

Reply:
I'm indifferent, I would probably just:

```lua
if redis.call("msetnx", unpack(KEYS)) == 0 then
  return 0
end

for _, key in ipairs(KEYS) do
  # ...
end
return 1
```

Reply:
Oh, this script needs some updating after the most recent PR #71

Reply:
I'v made necessary changes to the script. Please checkout obtain.lua (I renamed the script)